basics/plugin_structure/plugin_icon_file.md

```diff
@@ -1,42 +1,93 @@
+
+![36px x 36px is the area where the visible part of the logo icon should fit](img/icon_size.png){:width="225px"}
+
+Make sure Plugin Logos have the same visual weight as the logos in the examples below. 
+The more visual weight in Plugin Logo design, the less actual space it needs. 
+See more examples of [visual weight compensation](https://jetbrains.github.io/ui/principles/icons/#08) in the IntelliJ Platform UI Guidelines for Icons. 
+
+For basic shapes use the following sizes. 
+Note the different areas of transparent padding used for each shape: 
+
+| ![Square 32px x 32px](img/square_logo.png){:width="225px"} | ![Circle 34px in Diameter](img/circle_logo.png){:width="225px"} |
+|:---:|:---:|
+| _Square icon 32 x 32px_ | _Circular icon 34px in Diameter_ |
+| ![Horizontal rectangle 36px x 26px](img/rectangle_horizontal.png){:width="225px"} | ![Vertical rectangle 26px x 36px](img/rectancle_vertical.png){:width="225px"} |
+| _Horizontal rectangular icon 36px x 26px_ | _Vertical rectangular icon 26px x 36px_ |
+
+<br>
+
+### Plugin Logo Colors
+If the plugin’s technology already has a logo, use its colors. 
+Check the license terms before using the logo. 
+If there is no existing logo, or its use is prohibited, create a custom logo based on the [Action Colors Palette](https://jetbrains.github.io/ui/principles/icons/#action-icons) in the IntelliJ Platform UI Guidelines for Icons.
+
+| ![YouTrack Plugin Logo Uses the YouTrack Product Logo ](img/yt_logo.png){:width="225px"} | ![Keymap Plugin Logo Uses Color From the Action Colors Palette](img/keymap_logo.png){:width="225px"} |
+|:---:|:---:|
+| _The YouTrack Plugin Logo Uses<br>the YouTrack Product Logo_ | _The Keymap Plugin Logo Uses a Color<br>From the Action Colors Palette_ |
+
+Ensure a Plugin Logo is visible and provides a good user experience on both light and dark backgrounds. 
+If one Plugin Logo design does not work on both light and dark backgrounds, then create separate light and dark versions of the Plugin Logo. 
+The examples below illustrate how a good Plugin Logo design for a light background may work well (left) for a light background but not so well (middle) for a dark background. 
+Consequently, a separate Plugin Logo (right) for dark backgrounds is needed. 
+
+| ![Plugin Logo on Light UI Theme](img/light_version.png){:width="225px"} | ![Light Plugin Logo on Dark UI Theme](img/dark_bad.png){:width="225px"} | ![Plugin Logo for Dark UI Theme](img/dark_good.png){:width="225px"} |
+|:---:|:---:|:---:|
+| _The light Plugin Logo design<br>works well on light UI Theme_ | _The light Plugin Logo design does<br>not work well on a dark UI Theme_ | _A separate, dark Plugin Logo design<br>works well on dark UI Theme_ |
+
+### Plugin Logo Icon File Naming Convention
+To be used by the IntelliJ Platform, Plugin Logo icon files must be named according to the following conventions:
+* `pluginIcon.svg` as the default Plugin Logo, and specifically for use with light IDE UI Themes,
```

Comment:
This one is not only for light theme. It will be used for dark as well if there is no _dark verion. I think it will be more clear if we remove this part:
> and specifically for use with light IDE UI Themes,

Reply:
I think we need to clarify that in the event that a `pluginIcon_dark.svg` is supplied, the `pluginIcon.svg` file is specifically for the light IDE themes.